src/gpu/intel/ocl/tile_ops.h

```diff
@@ -196,6 +196,21 @@ DEF_BLOCK2D_LOAD_STORE(half, ushort, 16, 16, u16_m8k32v1, 32, 8)
 #define xlane_tile_access(t, i, j, sg, br, bc, nbr) \
     sub_group_broadcast(tile_access(t, i, j, sg, br, bc, nbr), i % sg)
 
+#define tile_predicated_assignment_t( \
+        t, sg_offset_r, sg_offset_c, f, value, sg, br, bc, nbr, nbc) \
+    do { \
+        for (int j = 0; j < (bc * nbc); j++) { \
```

Comment:
I'm still concerned an `int` type is used over a 64-bit analogue. I bet, @rjoursler has some insights on this regard.

Reply:
Somehow, I missed your response, I'm fine to address it some time later. Resolving...